tracer/build/_build/Build.Steps.cs

```diff
@@ -2772,11 +2773,34 @@ private async Task DownloadAndExtractVcpkg(AbsolutePath destinationFolder)
         EnsureExistingParentDirectory(destinationFolder);
         var parentFolder = destinationFolder.Parent;
 
-        CompressionTasks.UncompressZip(vcpkgZip, parentFolder);
+        UncompressZipQuiet(vcpkgZip, parentFolder);
 
         RenameDirectory(parentFolder / $"vcpkg-{vcpkgVersion}", destinationFolder.Name);
     }
 
+    // Quiet version of CompressionTasks:UncompressZip, which displays a log line for every created directory when uncompressing
+    private static void UncompressZipQuiet(string archiveFile, string directory)
+    {
+        Logger.Information("Uncompressing {File} to {Directory} ...", Path.GetFileName(archiveFile), directory);
+
+        using var fileStream = File.OpenRead(archiveFile);
+        using var zipFile = new ZipFile(fileStream);
+
+        var entries = zipFile.Cast<ZipEntry>().Where(x => !x.IsDirectory);
+        foreach (var entry in entries)
+        {
+            var file = PathConstruction.Combine(directory, entry.Name);
+            var path = Path.GetDirectoryName(file);
+
+            if (!Directory.Exists(path) && !string.IsNullOrEmpty(path))
+                Directory.CreateDirectory(path);
```

Comment:
coding style: we always use `{}` even for single-line blocks
```suggestion
                {
                    Directory.CreateDirectory(path);
                }
```